docs/source/60_building_apps.md

```diff
@@ -284,3 +284,327 @@ class MyCommandKryoRegistrator extends KryoRegistrator {
+applications should also depend on `adam-apis`. The main entrypoint to ADAM is
+the [ADAMContext](#adam-context), which allows genomic data to be loaded in to
+Spark as [GenomicRDD](#genomic-rdd). GenomicRDDs can be transformed using ADAM's
+built in [pre-processing algorithms](#algorithms), [Spark's RDD
+primitives](#transforming), and ADAM's [pipe](#pipes) APIs.
+
+### Accessing pre-built libraries
+
+These libraries are available from [Maven Central](http://search.maven.org) with
+the following coordinates:
+
+```
+<dependency>
+    <groupId>org.bdgenomics.adam</groupId>
+    <artifactId>adam-core${binary.version}</artifactId>
+    <version>0.21.0</version>
+</dependency>
+<dependency>
+    <groupId>org.bdgenomics.adam</groupId>
+    <artifactId>adam-apis${binary.version}</artifactId>
+    <version>0.21.0</version>
+</dependency>
+```
+
+For each release, we support four `${binary.version}`s:
+
+* `_2.10`: Spark 1.6.x on Scala 2.10
+* `_2.11`: Spark 1.6.x on Scala 2.11
+* `-spark2_2.10`: Spark 2.x on Scala 2.10
+* `-spark2_2.11`: Spark 2.x on Scala 2.11
+
+Additionally, we push nightly SNAPSHOT releases of ADAM to the
+[Sonatype snapshot repo](https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/),
+for developers who are interested in working on top of the latest changes in
+ADAM.
+
+### Loading data with the ADAMContext {#adam-context}
+
+The ADAMContext is the main entrypoint to using ADAM. The ADAMContext wraps
+an existing [SparkContext](http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.SparkContext)
+to provide methods for loading genomic data. In Scala, we provide an implicit
+conversion from a `SparkContext` to an `ADAMContext`. To use this, import the
+implicit, and call an `ADAMContext` method:
+
+```
+import org.apache.spark.SparkContext
+
+// the ._ at the end imports the implicit from the ADAMContext companion object
+import org.bdgenomics.adam.rdd.ADAMContext._
+import org.bdgenomics.adam.rdd.read.AlignmentRecordRDD
+
+def loadReads(filePath: String, sc: SparkContext): AlignmentRecordRDD = {
+  sc.loadAlignments(filePath)
+}
+```
+
+In Java, instantiate a JavaADAMContext, which wraps an ADAMContext:
+
+```
+import org.apache.spark.apis.java.JavaSparkContext;
+import org.bdgenomics.adam.apis.java.JavaADAMContext
+import org.bdgenomics.adam.rdd.ADAMContext;
+import org.bdgenomics.adam.rdd.read.AlignmentRecordRDD;
+
+class LoadReads {
+
+  public static AlignmentRecordRDD loadReads(String filePath,
+                                             JavaSparkContext jsc) {
+    // create an ADAMContext first
+    ADAMContext ac = new ADAMContext(jsc.sc());
+    
+    // then wrap that in a JavaADAMContext
+    JavaADAMContext jac = new JavaADAMContext(ac);
+
+    return jac.loadAlignments(filePath);
+  }
+}
+```
+
+With an `ADAMContext`, you can load:
+
+* Single reads as an `AlignmentRecordRDD`:
+  * From SAM/BAM/CRAM using `loadBam` (Scala only)
```

Comment:
If something says `(Scala only)` here, does that mean the method is not usable from Java or is simply inconvenient to do so?  I've been working with some of these from Java without issue.